filters/flowid/generator.go

```diff
@@ -0,0 +1,11 @@
+package flowid
```

Comment:
It doesn't bother me to move this to the filter.go file. I also thought how the entire set of Generator interface and implementations could be moved anywhere else. This was, basically, the reason why I already left it in this shape. Since we don't have any plans on moving them now, and they'll be exported afterwards, the interface might as well be moved to the filter.go file.